risc0/zkvm/src/guest/env.rs

```diff
@@ -97,45 +100,38 @@ pub(crate) fn finalize() {
     }
 }
 
-/// Exchange data with the host.
-pub fn send_recv_raw(channel: u32, to_host: &[u8], from_host: &mut [u32]) -> (u32, u32) {
+/// Exchange data with the host
+pub fn send_recv_raw(
```

Comment:
It looks like we are changing semantics to syscalls rather than send_recv. Will this function be deprecated/removed/renamed?

Reply:
Maybe this is `ioctl`?

Reply:
Or maybe just `syscall()`?